### PR TITLE
added tests for addition, subtraction, multiplcation, and division

### PR DESCRIPTION
**Pull request by rmzi**
Run with:
   
```(bash)
rspec spec/math_tests.rb
```

### CODE REVIEW

File: spec/math_tests.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import "math.rb"
+
+describe MathTest do
+
+  # Test Addition Function
+  describe "Addition" do
+    context "given x = 4 and y = 2" do
+      it "returns five" do
```
Comment (charleswisniewski) [inline spec/math_tests.rb]: not sure this returns five

**Review by charleswisniewski — COMMENTED** [review]
a few of the comments on this PR should be addressed

File: spec/math_tests.rb (hunk 1 above)
Comment (rmzi) [reply spec/math_tests.rb]: lol, addressed 🔢 🥇

**Comment by charleswisniewski** [comment]
LGTM :tada: :shipit:

**Pull request merged**